crates/test_utils/src/agent.rs

```diff
@@ -143,6 +163,15 @@ impl AgentBuilder {
         self
     }
 
+    /// Set the tombstone flag for the agent.
+    ///
+    /// This is a signal from this agent that it is planning to go offline or otherwise doesn't
+    /// want to be contacted anymore.
+    pub fn with_tombstone(mut self, is_tombstone: bool) -> Self {
```

Comment:
To not have to break the `AgentBuilder` later, should we add a `with_[...]` for all other remaining fields as well? `wit_agent()`, `with_space()`, `with_created_at()`, `with_expires_at()`. Could also be a separate PR but might be nice to do as part of the same release.

Reply:
https://github.com/holochain/kitsune2/pull/230/commits/846819386204d3f9f26ab367d8c02ea9d0f552b6